rust/connlib/clients/apple/build.rs

```diff
@@ -1,145 +1,15 @@
-// Referenced from https://github.com/chinedufn/swift-bridge/blob/master/examples/rust-binary-calls-swift-package/build.rs
```

Comment:
Why would reverting the build script work? The Rust library needs to link to the Swift library, since connlib needs to call the Swift-defined callback handler.

Reply:
The linking of libconnlib.a and the CallbackHandler happens as part of the main app's build, in this PR.

So, we don't need the Rust library linking to any Swift code at the time when build.rs is invoked.